README.adoc

```diff
@@ -3,7 +3,7 @@ ifdef::env-github,env-browser[:relfileprefix: docs/]
 
 https://travis-ci.org/CS2113-AY1819S1-W12-1/main[image:https://travis-ci.org/se-edu/addressbook-level4.svg?branch=master[Build Status]]
 https://ci.appveyor.com/project/damithc/addressbook-level4[image:https://ci.appveyor.com/api/projects/status/3boko2x2vr5cc3w2?svg=true[Build status]]
-https://coveralls.io/github/CS2113-AY1819S1-W12-1/main?branch=master[image:https://coveralls.io/repos/github/CS2113-AY1819S1-W12-1/main/badge.svg?branch=master[Coverage Status]]
+https://coveralls.io/github/jitwei98/main?branch=master[image:https://coveralls.io/repos/github/jitwei98/main/badge.svg?branch=master[Coverage Status]]
```

Comment:
did you commit the wrong file? cos it seems like you're changing the coveralls badge here

Reply:
yup will commit a correct version